_data/chains/eip155-4544.json

```diff
@@ -9,15 +9,15 @@
     "decimals": 18
   },
   "infoURL": "https://emoney.network/",
-  "shortName": "emoney",
+  "shortName": "EmoneyTestnet",
   "chainId": 4544,
   "networkId": 4544,
   "slip44": 118,
   "icon": "emoney",
   "explorers": [
     {
-      "name": "EMoney ethscan",
-      "url": "https://ethscan.emoney.network",
+      "name": "EMoney Explorer",
+      "url": "https://explore-stage.emoney.network/dashboard",
```

Comment:
this change makes it incompaible with EIP3091

Reply:
update is done, pls recheck. thanks